metadata/release/sample.json

```diff
@@ -104,10 +104,10 @@
         },
         "kubevirt": {
           "artifacts": {
-            "qcow2.xz": {
+            "ociarchive": {
               "disk": {
-                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.qcow2.xz",
-                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.qcow2.xz.sig",
+                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.ociarchive",
```

Comment:
I tried `cosa buildextend-kubevirt` followed by `cosa compress` and it created an `ociarchive.xz`.  If that is correct, we should append `.xz` to `ociarchive` in all the kubevirt sections here.

Reply:
I'm thinking we don't want an `ociarchive.xz` but rather a `.ociarchive` like we do for our other container artifacts we create. We'll probably need to update `cosa` to do the right thing here.

Reply:
Thanks for catching this @bgilbert

Reply:
Okay, in that case we'll want to drop the `uncompressed-sha256` here.